<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Helvetica</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>CBNU Grading Management System</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>561</width>
     <height>251</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>이름</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>학번</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>비밀번호</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>대학(원)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>학과(부)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>권한</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="lineEdit_search">
   <property name="geometry">
    <rect>
     <x>367</x>
     <y>124</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_search">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>120</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>검색</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_id">
   <property name="geometry">
    <rect>
     <x>213</x>
     <y>125</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>학번</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_name">
   <property name="geometry">
    <rect>
     <x>273</x>
     <y>125</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>이름</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>601</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>430</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>회원삭제</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-20</y>
     <width>671</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_edit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>정보수정</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_refresh">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>430</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>새로고침</string>
   </property>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>tableWidget</zorder>
  <zorder>lineEdit_search</zorder>
  <zorder>pushButton_search</zorder>
  <zorder>radioButton_id</zorder>
  <zorder>radioButton_name</zorder>
  <zorder>pushButton_cancel</zorder>
  <zorder>pushButton_delete</zorder>
  <zorder>pushButton_refresh</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
